### ISSUE TITLE
rviz crashes when motion planning is initialized in turtlebot3_home_service_challenge_demote_remote.launch - kinetic-devel

### ISSUE THREAD

**Issue by meard** [ROBOTIS-GIT/turtlebot3_home_service_challenge#23]
The branch for the repo is kinetic-devel.
When urtlebot3_home_service_challenge_demote_remote.launch is run, the rviz and navigation functionality works fine. However, when motion planning is initialized in rviz to operate openmanipulator, it shows no transform available between frame error and crashes. 
![Screenshot from 2022-03-04 01-29-42](https://user-images.githubusercontent.com/40901420/156676176-c28c6400-add4-48c7-9afe-4db661017a88.png)

**Comment by ruanjinchen** [comment]
I have the same problem, have you fixed it?

**Comment by meard** [comment]
Yes. The problem is primarily with the order in which Rviz display component initializes on launch. What helped me was
1. Disable all rviz display components in rviz and save it.
2. Enable autonomous driving display component (planner, global planner, local planner) and then motion planning after that ?
3. This should let you operate turtlebot3 with open manipulator and visualize it in rviz

**Comment by ruanjinchen** [comment]
Thank you for your reply. After I started rviz, this problem occurred. Is the solution you mentioned also applicable? And in my rviz, the model of the robot is not displayed normally, I don't know how to solve it, thank you again for your help.
![1](https://user-images.githubusercontent.com/102563903/162554653-408ab8cd-c3de-4c38-a6d9-81e4e3785cad.png)

**Comment by meard** [comment]
I did have this issue. Is you git branch kinetic-devel or master? Please make sure that you have the proper gt branch checked out. Also for me, this is mostly due to the warmup issue, where it takes a bit of time to populate all the robot model components in rviz. 
In the hindsight, check your tf tansform tree. That might give you an idea on where is the model going wrong.

**Comment by sunghowoo** [comment]
Since this issue has been inactive for a long time and appears to be resolved, I will proceed with closing it.
